### DISCUSSION TITLE
Add permission to limit who can publish site

### DISCUSSION THREAD

**Discussion by brittanisavery** [colibris-xyz/strapi-plugin-site-publisher#106] [Ideas]
Hi there,

I was wondering if it would be possible to add a plugin permission option to deny certain users from publishing the site. Maybe turning the READ permission on/off for it?

My dad and I are running a fictional basketball league and we would love to have others contributing articles about their teams and players. But, I would prefer to keep strapi as simple as possible for the guest contributors.

Thanks!

**Comment by tms0** [comment]
Hi @brittaniSavery,

Sorry I didn't use this plugin anymore and don't have time to invest on it for the moment. But I'm open to contribution if someone want to implement it.
